lib/airbrakex/notifier.ex

```diff
@@ -21,14 +21,24 @@ defmodule Airbrakex.Notifier do
         |> add_error(error)
         |> add_context(Keyword.get(options, :context))
         |> add(:session, Keyword.get(options, :session))
-        |> add(:params, Keyword.get(options, :params))
+        |> add(:params, filter_parameters(Keyword.get(options, :params), Config.get(:airbrakex, :filter_parameters, [])))
         |> add(:environment, Keyword.get(options, :environment, %{}))
         |> Jason.encode!()
 
       post(url(), payload, @request_headers, http_options())
     end
   end
 
+  defp filter_parameters(params, filtered_keys) do
+    (params || %{})
+      |> Enum.map(&filter_parameter(&1, filtered_keys))
+      |> Enum.into(%{})
+  end
+
+  defp filter_parameter({key, value}, filtered_keys) do
+    if Enum.member?(filtered_keys, Atom.to_string(key)), do: {key, "***"}, else: {key, value}
```

Comment:
Privilégier simplement `to_string` ici au lieu de `Atom.to_string`. Ça fait la même chose pour les atoms, et ça ne plantera pas si quelqu'un décide de mettre une string comme key dans la config.
```suggestion
    if Enum.member?(filtered_keys, to_string(key)), do: {key, "***"}, else: {key, value}
```